src/SFA.DAS.AssessorService.Data/RegisterRepository.cs

```diff
@@ -101,21 +101,21 @@ public async Task<string> CreateEpaOrganisationStandard(EpaOrganisationStandard
 
             if (null != organisationStandard.StandardVersions)
             {
-                foreach (var version in organisationStandard.StandardVersions)
+                foreach (var standardVersion in organisationStandard.StandardVersions)
                 {
-                    var standardUid = $"{organisationStandard.StandardReference.Trim()}_{version.Trim()}";
+                    var standardUid = $"{organisationStandard.StandardReference.Trim()}_{standardVersion.Version.Trim()}";
```

Comment:
Now we've changed the input from string Versions to the actual Organisation Standard Version object type. Is there any reason we can't supply the StandardUId rather than crafting it?